2-mandatory/1-create-functions.js

```diff
@@ -44,7 +49,14 @@ Write a function that:
 - Numbers greater 100 must be replaced with 100.
 */
 
-function formatPercentage() {
+function formatPercentage(arr) {
+  const formattedPercentages = numbers.map((number) => {
```

Comment:
```suggestion
  const formattedPercentages = arr.map((number) => {
```

numbers is not defined here and also your tests are not passing for this function. 

I'd suggest using `Math.round(num * 100) / 100` instead of `toFixed()`